src/util/color.cpp

```diff
@@ -25,7 +36,7 @@ void Color::Init(PresetColor c)
 
 void Color::Init(float red, float green, float blue)
 {
-    red_   = red;
-    green_ = green;
-    blue_  = blue;
```

Comment:
I also added clamping for the color values here - there isn't a generic clamp function available in libDaisy AFAICT so I yanked the templated version from `leddriver.h`

Reply:
weird. I could have sworn we at least had an assembly-optimized floating point clamp function in here, but I guess that's in the DaisySP side of things. 

Fine to leave it here for now, though!

Reply:
I will take another look just to be sure. There is one in DaisySP (in dsp.h) but I couldn't find anything equivalent in libDaisy.